### PR TITLE
ADEN-5439 | Disable admixes ABTest & enable admix3 on FV pages

### PR DESCRIPTION
**Pull request by kvas-damian**
https://wikia-inc.atlassian.net/browse/ADEN-5439

@Wikia/adeng

### CODE REVIEW

File: extensions/wikia/AdEngine/js/context/adContext.js [hunk 1]
```diff
@@ -88,23 +87,18 @@ define('ext.wikia.adEngine.adContext', [
...
 			context.targeting.skin === 'oasis' &&
+			context.targeting.hasFeaturedVideo &&
 			geo.isProperGeo(instantGlobals.wgAdDriverAdMixCountries)
 		);
 
-		context.opts.adMix1Enabled = context.opts.adMixExperimentEnabled && abTest.getGroup('AD_MIX').indexOf('AD_MIX_1') === 0;
-		context.opts.adMix3Enabled = context.opts.adMixExperimentEnabled && abTest.getGroup('AD_MIX').indexOf('AD_MIX_3') === 0;
-
-		context.slots.adMixToUnblock = [];
+		// TODO remove admix1 in ADEN-5443
```
Comment (WikiaTech) [inline extensions/wikia/AdEngine/js/context/adContext.js:97]: INFO Complete the task associated to this TODO comment. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1135)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 4 issues
* ![MAJOR][MAJOR] 3 major
* ![INFO][INFO] 1 info

Watch the comments in this conversation to review them.

#### 3 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [adContext.js#L69](https://github.com/Wikia/app/blob/885bc45be59385f1e2001f7651d5bb7954f78591/extensions/wikia/AdEngine/js/context/adContext.js#L69): Function has a complexity of 11 which is greater than 10 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AFunctionComplexity)
1. ![MAJOR][MAJOR] [adContext.js#L119](https://github.com/Wikia/app/blob/885bc45be59385f1e2001f7651d5bb7954f78591/extensions/wikia/AdEngine/js/context/adContext.js#L119): Function has a complexity of 31 which is greater than 10 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AFunctionComplexity)
1. ![MAJOR][MAJOR] [adContext.js#L197](https://github.com/Wikia/app/blob/885bc45be59385f1e2001f7651d5bb7954f78591/extensions/wikia/AdEngine/js/context/adContext.js#L197): Reduce the number of conditional operators (4) used in the expression (maximum allowed 3). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1067)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: extensions/wikia/AdEngine/js/context/adContext.js [hunk 2]
```diff
@@ -88,23 +87,18 @@ define('ext.wikia.adEngine.adContext', [
...
+		context.opts.adMix1Enabled = false;
+		context.opts.adMix3Enabled = context.opts.adMixExperimentEnabled;
 
-		if (!(context.opts.adMix1Enabled && context.targeting.hasFeaturedVideo)) {
-			context.slots.adMixToUnblock.push('INCONTENT_BOXAD_1');
-		}
+		context.slots.adMixToUnblock = ['INCONTENT_BOXAD_1'];
```
Comment (fraszczakszymon) [inline extensions/wikia/AdEngine/js/context/adContext.js]: Is it ok?

**Review by fraszczakszymon — APPROVED** [review]

**Comment by WikiaTech** [comment]
SonarQube analysis reported 4 issues
* ![MAJOR][MAJOR] 3 major
* ![INFO][INFO] 1 info

Watch the comments in this conversation to review them.

#### 3 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [adContext.js#L69](https://github.com/Wikia/app/blob/96f558edde5792144dc83dabaf84a944065fe7cc/extensions/wikia/AdEngine/js/context/adContext.js#L69): Function has a complexity of 11 which is greater than 10 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AFunctionComplexity)
1. ![MAJOR][MAJOR] [adContext.js#L119](https://github.com/Wikia/app/blob/96f558edde5792144dc83dabaf84a944065fe7cc/extensions/wikia/AdEngine/js/context/adContext.js#L119): Function has a complexity of 31 which is greater than 10 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AFunctionComplexity)
1. ![MAJOR][MAJOR] [adContext.js#L197](https://github.com/Wikia/app/blob/96f558edde5792144dc83dabaf84a944065fe7cc/extensions/wikia/AdEngine/js/context/adContext.js#L197): Reduce the number of conditional operators (4) used in the expression (maximum allowed 3). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1067)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Pull request merged**